### DISCUSSION TITLE
Why does slugify show an unpublished content?

### DISCUSSION THREAD

**Discussion by superpep** [ComfortablyCoding/strapi-plugin-slugify#7] [Q&A] [answered]
I was trying out slugify since I want to filter by a unique name and not by a default ID but I saw that it also shows the unpublished content... is this a bug or a feature? If it is a feature, can I disable it by any mean?

Example:
![Captura de pantalla de 2022-02-21 08-55-04](https://user-images.githubusercontent.com/37349590/154911831-cb641470-6883-4cea-b31d-a002cc82c357.png)

You can see it hasn't got a publish date since it hasn't been published

**Comment by ComfortablyCoding** [comment, marked as answer]
Hello @superpep ,

Thank you for the interest in this plugin!

At the moment it is intended as the only filter applied by the plugin is for the slug, any others are from the request being received. I believe the default behaviour should be not to show unpublished entities as that is inline with Strapis default. This was more of an oversight, I will include this in the next release. 

In the meantime adding the [publication state filter](https://docs.strapi.io/developer-docs/latest/developer-resources/database-apis-reference/rest/filtering-locale-publication.html#publication-state) will mimic this behaviour.

**Reply by superpep** [reply]
I'll try out the public state filter until you add this in the next release. I'll stay tuned for the release!

**Reply by ComfortablyCoding** [reply]
v2.0.0 which has this change has been released. You can view the full changelog [here](https://github.com/ComfortablyCoding/strapi-plugin-slugify/releases/tag/2.0.0)

**Reply by superpep** [reply]
Nice, just updated & tested it and works perfectly. Thank you.
